### ISSUE TITLE
Podcastindex phase 2 support

### ISSUE THREAD

**Issue by mpgirro** [mpgirro/stalla#49]
Phase 2 is now closed. See [documentation](https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/1.0.md) for additions.

**Comment by mpgirro** [comment]
Started implementing this in branch [podcastindex2](https://github.com/mpgirro/stalla/tree/podcastindex2)

**Comment by rock3r** [comment]
Is this done all in #90? Can we close this?

**Comment by mpgirro** [comment]
Yes this is all done in #90. Odd though, the PR had the closing keyword, which should have taken care of it.
